boards.txt

```diff
@@ -1711,6 +1711,20 @@ LoRa.menu.upload_method.dfuMethod.upload.tool=stm32CubeProg
 3dprinter.menu.pnum.VAKE_F446VE.build.variant=VAKE_F446VE
 3dprinter.menu.pnum.VAKE_F446VE.build.cmsis_lib_gcc=arm_cortexM4lf_math
 
+# FYSETC_S6 board
+3dprinter.menu.pnum.FYSETC_S6=FYSETC_S6
+3dprinter.menu.pnum.FYSETC_S6.upload.maximum_size=524288
```

Comment:
If you do not use the LD_* definition in the linker script then set the size with bootloader size less.
```suggestion
3dprinter.menu.pnum.FYSETC_S6.upload.maximum_size=458752
```